instrumentation/spring/spring-webflux/spring-webflux-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webflux/v5_3/SpringWebfluxTelemetry.java

```diff
@@ -46,6 +55,19 @@ public void addClientTracingFilter(List<ExchangeFilterFunction> exchangeFilterFu
         return;
       }
     }
-    exchangeFilterFunctions.add(new WebClientTracingFilter(instrumenter, propagators));
+    exchangeFilterFunctions.add(new WebClientTracingFilter(clientInstrumenter, propagators));
+  }
+
+  public TelemetryProducingWebFilter createWebFilter() {
```

Comment:
Maybe these should return `WebFilter` as `TelemetryProducingWebFilter` is package private.

Reply:
Yeah you're right. I've gone ahead and changed it to `WebFilter`.